DEVELOPER.MD

```diff
@@ -96,14 +97,26 @@ There are at least 7 containers that will be running:
       command: ` docker-compose  run --rm  pv_data_pipeline_initializer db upgrade`
 - Initializer container is expected to run and exit.
 
-### Flower, Initializer, Scheduler & Webserver
-
-- These are all based on `patentsview/airflow` container
-- The docker compose configuration differ only in two ways:
-    - The command is different (airflow scheduler v airflow webserver for example)
-    - Port numbers exposed are different
+### Celery Executor/ RabbitMQ / Flower
+- [CeleryExecutor]((https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/executor/celery.html)) (Task Queue software) is one of the ways you can scale out the number of workers. For this to work, you need to setup a Celery backend, with`RabbitMQ` (Message Broker), and change your airflow.cfg to point the executor parameter to CeleryExecutor and provide the related Celery settings.
```

Comment:
Can we go a step further and add screenshots of what the UI for these tools look like? Flower and RabbitMQ particularly?